swagger.yaml

```diff
@@ -5768,6 +5768,8 @@ definitions:
       - 'compatible-agent'
       - 'no-skip-installation-disk'
       - 'no-skip-missing-disk'
+      - 'service-can-connect-to-spoke-kube-api'
```

Comment:
Perhaps now that the validation is more specific it should be renamed - something like `service-has-sufficient-spoke-kube-api-access`